zio-http/src/main/scala/zhttp/service/Handler.scala

```diff
@@ -244,4 +271,13 @@ private[zhttp] final case class Handler[R](
       _ <- ChannelFuture.unit(ctx.writeAndFlush(LastHttpContent.EMPTY_LAST_CONTENT))
     } yield ()
   }
+
+  private def unsafeWriteFileContent(path: java.nio.file.Path)(implicit ctx: ChannelHandlerContext) = {
+    import java.io.RandomAccessFile
+    val file       = new File(path.toString)
+    val raf        = new RandomAccessFile(file, "r")
+    val fileLength = raf.length()
+    ctx.write(new DefaultFileRegion(raf.getChannel, 0, fileLength))
```

Comment:
Can do it here or create a separate issue for handling range requests. Such that the range is not hardcoded to `0-fileLength`.

Reply:
I'll do this separately -  [#709](https://github.com/dream11/zio-http/issues/709)